### PR TITLE
[34] Disable broken /stats endpoint.

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: engine/src/main/scala/cromwell/webservice/CromwellApiService.scala [hunk 1]
```diff
@@ -60,11 +59,7 @@ trait CromwellApiService extends HttpInstrumentation {
   val engineRoutes = concat(
     path("engine" / Segment / "stats") { _ =>
       get {
-        onComplete(workflowManagerActor.ask(WorkflowManagerActor.EngineStatsCommand).mapTo[EngineStatsActor.EngineStats]) {
-          case Success(stats) => complete(ToResponseMarshallable(stats))
-          case Failure(e: TimeoutException) => e.failRequest(StatusCodes.ServiceUnavailable)
-          case Failure(_) => new RuntimeException("Unable to gather engine stats").failRequest(StatusCodes.InternalServerError)
-        }
+        completeResponse(StatusCodes.Forbidden, APIResponse.fail(new RuntimeException("The /stats endpoint is currently disabled.")), warnings = Seq.empty)
```
Comment (cjllanwarne) [inline engine/src/main/scala/cromwell/webservice/CromwellApiService.scala]: Do you really need the overhead of creating an instantly-unpacked exception here? (ie is there not just a "use this message" version of the method?)

Also, it's not really a `RuntimeException`...

File: engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala [hunk 2]
```diff
@@ -36,7 +36,7 @@ class CromwellApiServiceSpec extends AsyncFlatSpec with ScalatestRouteTest with
 
   implicit def default = RouteTestTimeout(5.seconds)
 
-  "REST ENGINE /stats endpoint" should "return 200 for stats" in {
+  "REST ENGINE /stats endpoint" should "return 200 for stats" ignore {
```
Comment (danbills) [inline engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala]: could delete instead?

**Comment by ruchim** [comment]
👍

File: engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala (hunk 2 above)
Comment (ruchim) [reply engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala]: do we ever want to bring this back?

Comment (danbills) [reply engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala]: mm yeah, we do.  nm

**Pull request merged**